components/remote_settings/src/lib.rs

```diff
@@ -46,16 +46,24 @@ pub struct RemoteSettingsService {
 impl RemoteSettingsService {
     /// Construct a [RemoteSettingsService]
     ///
-    /// This is typically done early in the application-startup process
+    /// This is typically done early in the application-startup process.
+    ///
+    /// This method performs no IO or network requests and is safe to run in a main thread that
```

Comment:
This part of the comment is wrong now, right? It *does* (well, "may") perform IO and thus is *not* safe to call on the main thread.

Reply:
I think it's right.   There's no IO preformed until we actually need to use the DB, which doesn't happen until something like `get_records` is called (https://github.com/mozilla/application-services/blob/42a75dec389e0a0673bc2836fdaee9d8d64a7a5e/components/remote_settings/src/storage.rs#L44).